oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/ProcessServiceLabel.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core.source;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.skywalking.oap.server.core.Const;
+import org.apache.skywalking.oap.server.core.analysis.IDManager;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+import static org.apache.skywalking.oap.server.core.source.DefaultScopeDefine.PROCESS_SERVICE_LABEL;
+
+@ScopeDeclaration(id = PROCESS_SERVICE_LABEL, name = "ProcessServiceLabel")
+@ScopeDefaultColumn.VirtualColumnDefinition(fieldName = "entityId", columnName = "entity_id", isID = true, type = String.class)
+public class ProcessServiceLabel extends Source {
```

Comment:
Where is the label data's timestamp? I think label should have TTL too, right?

Reply:
Also, I think ServiceLabels is a better name? ProcessServiceLabel makes me feeling strange whether this is a service or process's label.

Reply:
> Where is the label data's timestamp? I think label should have TTL too, right?

Yes, the `timeBucket` is already in the `Source`, so I just use this field.

Reply:
> Also, I think ServiceLabels is a better name? ProcessServiceLabel makes me feeling strange whether this is a service or process's label.

Updated.